src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +15,33 @@ public MovieDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
```

Comment:
why not `try-with-resources`?

Reply:
Bc Transaction doesn't implement AutoClosable, therefore I will need to use finally block to close it anyway. With this I decided to use finally for both cases. Or is there any other way aruond?